### PR TITLE
liquidsoap-daemon.0.1.2

### PR DESCRIPTION
**Pull request by toots**

### CODE REVIEW

**Comment by yallop** [comment]
The build's failing because `duppy.0.5.0` doesn't build without camlp4.  See #8602.

**Comment by yallop** [comment]
It's now hitting [Mantis 7426](https://caml.inria.fr/mantis/view.php?id=7426):

```
# >> Fatal error: Assignment of a float to a specialised non-float array: (array.unsafe_set[addr]<{audio.ml:668,1-10}>
#                                                           self-4/3888
#                                                           vol/3886
#                                                           Parraysetu_arg/3892)
```

File: packages/liquidsoap-daemon/liquidsoap-daemon.0.1.2/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+install: [
+  ["./daemonize-liquidsoap.sh"] { os = "linux" }
+  ["env" "init_type=launchd" "./daemonize-liquidsoap.sh"] { os = "darwin" }
+]
+remove: [
+  ["env" "mode=remove" "./daemonize-liquidsoap.sh"] { os = "linux" }
+  ["env" "mode=remove" "env" "init_type=launchd" "./daemonize-liquidsoap.sh"] { os = "darwin" }
+]
+depends: [
+  "liquidsoap"
```
Comment (avsm) [inline packages/liquidsoap-daemon/liquidsoap-daemon.0.1.2/opam]: I think you're missing a dependency on `camlp4 {build}` here

```
#=== ERROR while compiling duppy.0.5.0 ========================================#
# command              make
# path                 /home/opam/.opam/ocaml-base-compiler.4.03.0/.opam-switch/build/duppy.0.5.0
# exit-code            2
# env-file             /home/opam/.opam/log/duppy-31-de96f2.env
# output-file          /home/opam/.opam/log/duppy-31-de96f2.out
### output ###
# make -C src all
# make[1]: Entering directory '/home/opam/.opam/ocaml-base-compiler.4.03.0/.opam-switch/build/duppy.0.5.0/src'
# ocamlfind ocamlc -c -package camlp4 -package camlp4.quotations.o -package camlp4.extend -syntax camlp4o pa_duppy.mli
# ocamlfind: Package `camlp4.quotations.o' not found
# Makefile:50: recipe for target 'pa_duppy.cmi' failed
# make[1]: *** [pa_duppy.cmi] Error 2
# make[1]: Leaving directory '/home/opam/.opam/ocaml-base-compiler.4.03.0/.opam-switch/build/duppy.0.5.0/src'
# Makefile:12: recipe for target 'all' failed
# make: *** [all] Error 2

```

Comment (yallop) [inline packages/liquidsoap-daemon/liquidsoap-daemon.0.1.2/opam]: @avsm: that should be fixed by #8602

Comment (avsm) [inline packages/liquidsoap-daemon/liquidsoap-daemon.0.1.2/opam]: Thanks @yallop!  If you could push a merge commit from opam-repository:master to your branch @toots, the CI should retest with the #8602 fix included.

**Comment by toots** [comment]
Awesome guys, thanks for taking care of this.

**Pull request merged**